scripts/oauth/token.go

```diff
@@ -272,7 +272,7 @@ func setup() *OAuthApp {
 
 	substitutions, err := registry.GetMap(SubstitutionsFieldName)
 	if err != nil {
-		slog.Warn("no substitutions, ensure that the provider info doesn't have any {{variables}}", "error", err)
+		slog.Warn("no substitutions, ensure that the provider info doesn't have any variables", "error", err)
```

Comment:
I understand why this line was changed as I had troubles myself. But the brackets have to stay as it is referes to catalog variables, ex: `{{.workspace}}`.